src/Adapter/EntityMapper.php

```diff
@@ -87,7 +87,7 @@ public function load($id, $id_lang, $entity, $entity_defs, $id_shop, $should_cac
                 }
                 $entity->id = (int)$id;
                 foreach ($object_datas as $key => $value) {
-                    if (array_key_exists($key, $entity)) {
+                    if (array_key_exists($key, $entity_defs['fields'])) {
```

Comment:
this is the **** of adding a field to Product|Customer|... entity: we have to define public property in the class when in PHP it's totally valid (and fast) to define public properties dynamically.